tests/test_add_new_todo.py

```diff
@@ -1,24 +1,33 @@
+import argparse
+import sys
+import os
+
+myPath = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, myPath+'/../')
+
+from core.webdriver_factory import WebdriverFactory
 from typing import List
 from core.todo import Todo
 from core.web_framework import web_framework
+from screens.screen_factory import screen
+from core.verify import verify
 
 
 @web_framework
 def test_create_new_todo(driver):
-    '''new_todo: Todo = Todo(text='Hello Blat')
-    create_new_todo(text=new_todo_text)
-    exisiting_todos: List[Todo] = get_existing_todos()  ?зачем
-    new_todo_exists = new_todo in exisiting_todos
-    verify(param_1=new_todo_exists, param_2=True, operator='==')  ? не понимаю способа проверки'''
-    to_do = Todo('Hello Blat', 'June 05 2021', True)
-    create_new_todo(driver)
-    existing_todos: List = get_existing_todos(driver)
-    assert verify([to_do.new_todo_text, to_do.new_todo_due_date], existing_todos[0])
+    to_do = Todo(text='Hello Blat', due_date='Wed Jul 05 2023')
+    # to_do = Todo(text='Hello Blat')
+    screen('todo').create_new_todo(to_do, driver)
+    existing_todos: List = screen("todo").get_existing_todos(driver)
+    assert verify(arg_1=to_do.text in existing_todos[0].text, arg_2=True, operator='==')
```

Comment:
it has to be like this: arg1 = todo in existing todos
you'll need to implement  __eq__ in Todo class